app/gui/view/graph-editor/src/component/node/input/area.rs

```diff
@@ -318,250 +320,12 @@ impl Model {
             port.set_usage_type(tp)
         }
     }
-}
-
-fn select_color(styles: &StyleWatch, tp: Option<&Type>) -> color::Lcha {
-    let opt_color = tp.as_ref().map(|tp| type_coloring::compute(tp, styles));
-    opt_color.unwrap_or_else(|| styles.get_color(theme::code::types::any::selection).into())
-}
-
-
-
-// ============
-// === Area ===
-// ============
-
-/// Input ports area.
-///
-/// ## Origin
-/// Please note that the origin of the node is on its left side, centered vertically. To learn more
-/// about this design decision, please read the docs for the [`node::Node`].
-#[derive(Clone, CloneRef, Debug)]
-pub struct Area {
-    #[allow(missing_docs)]
-    pub frp:          Frp,
-    pub(crate) model: Rc<Model>,
-}
-
-impl Deref for Area {
-    type Target = Frp;
-    fn deref(&self) -> &Self::Target {
-        &self.frp
-    }
```

Comment:
Why this code was moved below? Now it is inconsistent - FRP definition is under label "model" while it is not part of Model, its part of Area.

Reply:
Moved the FRP down, closer to the Area